lib/capistrano/tasks/sidekiq.rake

```diff
@@ -283,4 +283,12 @@ namespace :sidekiq do
       properties.fetch(:run_as) || # global property across multiple capistrano gems
       role.user
   end
+
+  def options_for_server(server, process_options = fetch(:sidekiq_options_per_process))
+    options = nil
+    server.roles.each do |role|
+      options = process_options[role] if process_options[role]
+    end
```

Comment:
I think this would behave incorrectly if one server had multiple roles with sidekiq options

Reply:
I see now that you are trying to send nil if there are none matching, maybe you could do that on the last line? That way we can still sum all of the possible sidekiq options from all of the roles.

Reply:
Updated to account for multiple roles